### DISCUSSION TITLE
云边通信，边缘节点被入侵后，怎么确保云端节点的安全

### DISCUSSION THREAD

**Discussion by smart666666** [kubeedge/kubeedge#5805] [Q&A]
目前看edgemesh-agent的配置，边缘和云端的地位是相同的，对于用于边缘智能网关的场景，一台网关作为边缘节点被入侵的时候，如何确保云端及其他网关的服务的api不被恶意调用；以及ssh代理的安全问题

**Comment by WillardHu** [comment]
EdgeMesh services edge apps(pod to pod), so I think the communication security needs to be ensured by user apps. BTW, EdgeMesah can add a tls for p2p connection.
